### PR TITLE
docs: show version switcher at tob in mobile

### LINKED ISSUE #614: Show version dropdown at top for mobile
https://github.com/GoogleCloudPlatform/gcloud-node/pull/612#issuecomment-103666623

### PR DESCRIPTION
**Pull request by stephenplusplus**
Fixes #614 

Preview: http://stephenplusplus.github.io/gcloud-node-again-again/#/docs/master

screen shot 2015-06-17 at 10 30 58 am

### CODE REVIEW

File: docs/site/components/subpage/subpage-directive.js [hunk 1]
```diff
@@ -23,6 +23,7 @@ angular
           $http.get('https://api.github.com/repos/GoogleCloudPlatform/gcloud-node/commits?sha=gh-pages&per_page=1')
             .then(function(resp) {
               $scope.lastBuiltDate = moment(resp.data[0].commit.committer.date).format('MMMM Do, YYYY');
+              $scope.lastBuiltDateSm = moment(resp.data[0].commit.committer.date).format('M/D/YY');
```
Comment (jgeewax) [inline docs/site/components/subpage/subpage-directive.js]: Could we factor this apart a bit?

```javascript
var $scope.lastBuildDate = moment(resp.data[0].commit.committer.date);
```

And then in the template do `{{lastBuildDate.format('...')}}` ?

Comment (stephenplusplus) [inline docs/site/components/subpage/subpage-directive.js]: I like that idea!

Comment (stephenplusplus) [inline docs/site/components/subpage/subpage-directive.js]: Fixed.

**Comment by jgeewax** [comment]
LGTM

**Pull request merged**